moped-editor/src/views/projects/projectView/ProjectTimelineViewQueryConf.js

```diff
@@ -0,0 +1,89 @@
+export const ProjectTimelineViewQueryConf = {
+  options: {
+    useQuery: {
+      fetchPolicy: "no-cache", // Default ise "cache-first", or use "no-cache"
+      // Use no-cache as per issue https://github.com/cityofaustin/atd-data-tech/issues/4788
```

Comment:
The update to Apollo fixed this issue for the staff and project queries so I took out this comment in those configs. I know that no cache is the behavior that we will want in some places so just giving an update.